snuba/cli/cleanup.py

```diff
@@ -13,7 +13,7 @@
               help="If true, only print which partitions would be dropped.")
 @click.option('--database', default='default',
               help='Name of the database to target.')
-@click.option('--table', default=settings.DEFAULT_LOCAL_TABLE,
+@click.option('--table',
```

Comment:
nit: Seems like the others take a dataset and then use it to get the `LOCAL_TABLE`, is this different for a reason?

Reply:
Oops, no this was just missed,